apps/server/src/services/ElasticSearchClient.ts

```diff
@@ -48,24 +49,39 @@ export async function reIndexElasticSearch() {
   } catch {
     info('Elasticsearch "institutions" index did not exist')
   }
-  info('Elasticsearch indexing institutions')
-  const dataFilePath = getInstitutionFilePath()
-  const rawData = readFileSync(dataFilePath)
-  const jsonData = JSON.parse(rawData.toString())
+  const institutionData = await getCachedInstitutions()
```

Comment:
The name getCachedInstitutions is making me think that this is coded wrong, because it's pulling institutions from the file.

Reply:
It's getting the institutions from either the cached file or the institution server `/cacheList` endpoint. Maybe `getCacheInstitutions ` or `getAvailableInstitutions` idk

Reply:
getInstitutions